scrapper_helpers/utils.py

```diff
@@ -1,23 +1,96 @@
 import os
 import pickle
+import subprocess
+import unicodedata
+
+try:
+    from __builtin__ import unicode
+except ImportError:
+    unicode = lambda x, *args: x
 
 DEBUG = os.environ.get('DEBUG')
 CACHE_DIR = os.environ.get('CACHE_DIR', '/var/tmp/scrapper-helpers/')
+MAX_FILENAME_LENGTH = os.environ.get('MAX_FILENAME_LENGTH', subprocess.check_output("getconf NAME_MAX /", shell=True))
```

Comment:
python3
```
>>> subprocess.check_output("getconf NAME_MAX /", shell=True)
b'255\n'
```
python2
```
>>> subprocess.check_output("getconf NAME_MAX /", shell=True)
'255\n'
```
so it needs `int(subprocess.check_output("getconf NAME_MAX /", shell=True).strip())`